### DISCUSSION TITLE
Tauri build wasm mime type

### DISCUSSION THREAD

**Discussion by TrueDoctor** [tauri-apps/tauri#5748] [Q&A]
When building my vue app using `cargo tauri build`, I recive this error upon startup of the app: 
```
Unhandled Promise Rejection: TypeError: Unexpected response MIME type. Expected 'application/wasm'
```
It seems like the the `.wasm` file is served with the wrong mime type. (It dev mode, everything works fine)
I tried to poke around the codebase and added the wasm type to mime_types in tauri utils, but I couldn't seem to be able to find 
where the bundled files are loaded in tauri. Any help would be appreciated

**Comment by JonasKruckenberg** [comment]
It would be great if you could file a proper issue for this and also fill out the required fields there!

**Reply by TrueDoctor** [reply]
Alright, created an issue #5749
